docs/specifications/tests/Delegation-TP/delegation01.md

```diff
@@ -1,48 +1,98 @@
-## DELEGATION01: Minimum number of name servers   
+# DELEGATION01: Minimum number of name servers   
 
-### Test case identifier
+## Test case identifier
 
 **DELEGATION01:** Minimum number of name servers
 
-### Objective
+## Objective
 
-Section 4.1 of [RFC 1034](https://tools.ietf.org/html/rfc1034) specifies that
-there must be at least minimum two name servers for a domain. This test is
-done to verify this condition
+Section 4.1 of [RFC 1034] specifies that there must be at least minimum two name servers 
+for a domain. This test is done to verify this condition.
 
-### Inputs
+The RFC ([RFC 1034]) pre-dates IPv6. Sinces IPv4 and IPv6 work as separate networks, this
+test case has been extended to test for for two name servers that resolv into IPv4 addresses
+and IPv6 addresses respectively.
 
-The domain name to be tested.
 
-### Ordered description of steps to be taken to execute the test case
-
-1. Obtain the complete set of name servers from parent using
-   [Method2](../Methods.md).
-2. If the number of name servers (distinct NS records) returned in previous step is less than two, the
-   record as an error.
-3. Obtain the complete set of name servers from the child 
-   using [Method3](../Methods.md).
-4. If the number of name servers (distinct NS records) returned in the previous step is less than two, the
-   record as an error.
-5. If an error was recorded in step 2 or step 4, then the test case fails.
- 
-### Outcome(s)
+## Inputs
 
-If the total amount of name servers used for the domain are two or more,
-then the test case succeeds.
+The domain name to be tested.
 
-### Special procedural requirements
+## Ordered description of steps to be taken to execute the test case
+
+1. Obtain the complete set of name servers (distinct NS records) from 
+   parent using [Method2](../Methods.md).
+2. If the number of name servers from parent is at least two emit the
```

Comment:
Would you consider adding a comma after the condition? That way the reader gets a visual hint about the structure of the sentence.

In some places there are comma signs before "else" too. I find those helpful as well.